R/sim_linelist.R

```diff
@@ -0,0 +1,217 @@
+#' Simulate a line list
+#'
+#' @description The line list is simulated using a branching process and
+#' parameterised with previously published epidemiological parameters.
```

Comment:
```suggestion
#' parameterised with user defined epidemiological parameters.
```

Reply:
Resolved in bee12c28a0a4cb009c64c7aa18bd169ef679bfd9.